zebra-consensus/src/transaction.rs

```diff
@@ -454,6 +468,12 @@ where
 
             let legacy_sigop_count = cached_ffi_transaction.legacy_sigop_count()?;
 
+            // TODO: Return an error for Request::Block as well to replace this check in
```

Comment:
This needs to be a WONTFIX because ticket #2336 is closed, see our new process in:
- #6281

```suggestion
            // WONTFIX: Return an error for Request::Block as well to replace this check in
```

Reply:
Applied in https://github.com/ZcashFoundation/zebra/pull/6510/commits/e6c6c0f33593095568ffef531379d445f2baa290.